src/model.cpp

```diff
@@ -0,0 +1,250 @@
+#include <model.hpp>
+
+namespace nest {
+namespace mc {
+
+// Helper function for building cell groups used by model constructor
+// See bottom of file for definition.
+cell_group_ptr make_cell_group(
```

Comment:
SHould this be on the cell_group interface. It has nothing to do with the model ( the model uses it.)
You dont expect it on this source location